unstable/lens/struct-list.scrbl

```diff
@@ -0,0 +1,23 @@
+#lang scribble/manual
+
+@(require lens/private/doc-util/main)
+
+@title{Converting between structs and lists}
+
+@deftogether[[
+  @defform[(struct->list-lens struct-id)]
+  @defform[(list->struct-lens struct-id)]]]{
+Lenses that convert between structs and lists.
```

Comment:
We should link to `struct->list` and describe what happens in the `on-opaque?` case (we just throw an error right?)